include/af/defines.h

```diff
@@ -496,6 +503,9 @@ namespace af
 #if AF_API_VERSION >= 34
     typedef af_random_engine_type randomEngineType;
 #endif
+#if AF_API_VERSION >= 35
+    typedef af_canny_threshold CannyThreshold;
```

Comment:
@9prady9 what do you think about making this typedef lower camel case to stay consistent with the surrounding code?